linux/help/about/whatsnew.md

```diff
@@ -2,7 +2,7 @@
 title: What's New
 ---
 
-Here are some of the new features we have added to Keyman for Linux 15.0:
+Here are some of the new features we have added to Keyman for Linux 16.0:
 
 * Open a .kmp file with Keyman Config (#3183)
```

Comment:
This one probably should've been left as is.

Reply:
Darcy wrote:

> We'll update whatsnew for each platform separately.

So it was right to update the titles now, and we'll go back and correct the dot points as separate PRs.

Reply:
Except the text that follows is for the _old_ version.  We should add a _new_ section with the points corresponding to the new version, rather than "edit" history to say that 15.0 things are new for 16.0.

Reply:
What's New only includes content for the latest version.